examples/basic-node/app.js

```diff
@@ -0,0 +1,12 @@
+var dust = require('dustjs-linkedin');
+
+var tmpl = dust.compile("Hello world! Using Dust version {version}!", "hello");
+dust.loadSource(tmpl);
```

Comment:
you're right. CJS loading would have been nice.  There's `compileFn` and `renderSource` but both of those use `loadSource` underneath.